pkg/api/encode_prom.go

```diff
@@ -17,11 +17,18 @@
 
 package api
 
+type PromTLSConf struct {
+	Enable   bool   `yaml:"enable,omitempty" json:"enable,omitempty" doc:"set to true to enable tls for the prometheus endpoint"`
```

Comment:
In kafka/loki we don't use an `enable` flag, we use a pointer instead and check for nil. (I know we can't do that in the operator, but here it's different)
So here also I think we should be consistent

Reply:
Changed it,
Thanks